.github/workflows/test.yml

```diff
@@ -0,0 +1,88 @@
+name: KBase Catalog test
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main or develop
+    branches:
+      - main
+      - master
+      - develop
+
+jobs:
+  catalog_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - python-version: '3.7'
+            mongo-version: '3.6'
+          - python-version: '3.7'
+            mongo-version: '7.0.4'
+    services:
+      mongo:
+        image: mongo:${{matrix.python-version}}
+        ports:
+            - 27017:27017
+        options: --name mongo${{matrix.python-version}}
+
+    steps:
+      - name: Repo checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{matrix.python-version}}
+
+      - name: Install dependencies and set up test config
+        shell: bash
+        env:
+          KBASE_CI_TOKEN: ${{ secrets.KBASE_CI_TOKEN }}
+          ADMIN_USER: kbase_bot
+
+        run: |
+
+          # test mongo connection
+          curl http://localhost:27017
+          returncode=$?
+          if [ $returncode != 0 ]; then exit $returncode; fi
+
+          # set HOMEDIR
+          export HOMEDIR=`pwd`
+
+          # move to parent dir to install binaries etc
+          cd ..
+
+          # setup kb-sdk
+          mkdir -p $(pwd)/bin
+          docker run kbase/kb-sdk genscript > $(pwd)/bin/kb-sdk
```

Comment:
This is the old sdk; I'm surprised it works here

Reply:
It is similar to https://kbase.github.io/kb_sdk_docs/tutorial/2_install.html

For setting up, how is the new SDK different from the old one?

Reply:
Yeah, that tutorial probably won't work in a lot of scenarios

The SDK patch Dakota made upgrades the docker service

Reply:
Is there anywhere I can find the latest setup/install instructions for kb-sdk?

Reply:
https://github.com/kbase/kb_sdk_patch

If it works it works though

Reply:
I switched to the patch, and it seems to be working.